util/check_dif_statuses.py

```diff
@@ -350,12 +416,19 @@ def main(argv):
         "--show-unimplemented",
         action="store_true",
         help="""Show unimplemented functions for each incomplete DIF.""")
+    parser.add_argument("--show-alerts",
+                        action="store_true",
+                        help="""Show alert functions for each DIF.""")
+    parser.add_argument("--show-irqs",
+                        action="store_true",
+                        help="""Show IRQ functions for each DIF.""")
```

Comment:
I wonder if it is more useful to show these by default (given the user is more likely interested in knowing what's not implemented yet) and have these switches behave in the reverse way instead.

Reply:
<img width="318" alt="Screen Shot 2021-08-26 at 10 24 29 AM" src="https://user-images.githubusercontent.com/5633066/132450574-fdd5d2ef-39a6-4dc2-997a-ce6c0d355392.png">

I think eventually this makes sense, but right now the main purpose of these switches is to enable me to print an _extra_ table (see above) to identify similar IRQ/Alert DIFs between IPs so I can design/implement a shared set of IRQ/Alert handler DIFs. Once this has been accomplished, I envisioned further editing this tool to print whether each shared IRQ/Alert DIF has been integrated into each IP's DIF set. Does this make sense?

Reply:
I see, okay.